src/compressed_tensors/quantization/lifecycle/apply.py

```diff
@@ -125,13 +125,14 @@ def apply_quantization_config(model: Module, config: QuantizationConfig) -> Dict
             target_to_scheme[target] = scheme
 
     # list of submodules to ignore
-    ignored_submodules = []
+    ignored_submodules = defaultdict(list)
```

Comment:
How does the dict fix this? It looks like we still cast to a set later on, how does that work?

Reply:
before:
it was comparing the size of the list.

after:
it compares the length of keys

Reply:
keys are the input users gave to ignore ex
ignore: ["lm_head", "re:.*gate"]